src/storage/exec/IndexOutputNode.h

```diff
@@ -276,7 +276,7 @@ class IndexOutputNode final : public RelNode<T> {
 
     void addVidToRow(VertexIDSlice vId, Row& row) {
         if (planContext_->isIntId_) {
-            row.emplace_back(vId.toString());
+            row.emplace_back(*reinterpret_cast<const int64_t*>(vId.data()));
```

Comment:
So vid will be explained as a string previously?

Reply:
Still need to copy, this is to prevent the id is more than expected 8 bytes.